app/javascript/packs/application.js

```diff
@@ -8,6 +8,9 @@ require("turbolinks").start()
 require("@rails/activestorage").start()
 require("channels")
 
+require("jquery3")
```

Comment:
Are you actually using these or just adding them in for bootstrap?

Reply:
This was just added for bootstrap.